sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -728,11 +732,57 @@ export class ApplicationStudentsController extends BaseController {
         `Application id ${applicationId} not found or not in relevant status to get enrolment details.`,
       );
     }
+
+    // Verify Disability Status PD/PPD.
+    const verifiedDisabilityStatus = application.currentAssessment.workflowData
+      .calculatedData.pdppdStatus
+      ? [(DisabilityStatus.PD, DisabilityStatus.PPD)].includes(
+          application.student.disabilityStatus,
+        )
+      : true;
```

Comment:
I will recommend applying centralization as much as possible to be in sync with e-cert logic.